blaze/expr/datetime.py

```diff
@@ -119,6 +129,27 @@ def utcfromtimestamp(expr):
     return UTCFromTimestamp(expr)
 
 
+class nanosecond(DateTime): _dtype = datashape.int64
```

Comment:
It turns out the mismatch is caused by https://github.com/blaze/odo/blob/master/odo/backends/pandas.py#L25, were `datetime_` is detected as "possibly_missing". What is the rationale for this ?
Should I adjust the failing test (by marking the expected schema as '?int64'), or does this require fixing in odo ?
(It would be good to have at least a bit of inline comments or even documentation in odo's `dshape_from_pandas` to help people understand the intent...)